tt_llk_blackhole/common/inc/sfpu/ckernel_sfpu_reduce.h

```diff
@@ -42,6 +38,34 @@ constexpr uint32_t ROWS_PER_TILE = 64;
 // ============================================================================
 // Helper Functions
 // ============================================================================
+/**
+ * @brief Apply sign magnitude conversion to four LREGs
+ * Assume you loaded 4 LREGs into p_sfpu::LREG0-3 and want to convert them to sign magnitude and store in p_sfpu::LREG4-7.
+ * @tparam INSTR_MOD_CAST The instruction mode for cast operations
+ */
+template <InstrModCast INSTR_MOD_CAST>
+inline void apply_sign_magnitude_conversion_four_lregs()
```

Comment:
```suggestion
inline void apply_sign_magnitude_x4()
```
Maybe, not sure about this one. WDYT @nvelickovicTT ?